libraries/ms-common/inc/bootloader_can.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+// This module allows the sending and receiving of CAN messages in the bootloader's CAN ID format.

```

Comment:
You should link the bootloader design doc Confluence page here (short link: https://uwmidsun.atlassian.net/l/c/76h1GfbH).